README.md

```diff
@@ -74,8 +74,24 @@ them together into one.
 Maintainers
 -------------------
 
-The project is maintained by
- - Nicco Kunzmann ([@niccokunzmann](https://github.com/niccokunzmann))
- - Tarun Kumar ([@meets2tarun](https://github.com/meets2tarun))
- 
- [gitter]: https://gitter.im/fossasia/open-event-orga-server
+You can reach the maintainers,
+ping them personally by looking at the [Badgeyay team][team].
+
+You can become a maintainer by following the project and contributing code to it.
+Please see your role in the [CCCC][cccc].
+
+The project is maintained by the [Badgeyay maintainer's team][team].
+To join the team:
+1. Contribute
+2. You or someone else proposes you in an issue to become a member of the team.
+3. A Badgeyay admin adds you.
+
+To be excluded from the team:
+1. Follow the rules of [CCCC][cccc] or [Badgeyay](.github/CONTRIBUTING.md) and do not violate them willingly or in a harmful way. 
```

Comment:
you can represent the first point as
`Do not violate the rules of .... or .... in  a harmful way`

Reply:
oh...